goal_src/jak1/engine/game/task/game-task-h.gc

```diff
@@ -127,5 +127,10 @@
   (plunger-lurker-hit 113)
   (leaving-misty 114)
   (assistant-village3 115)
-  (max 116)
+  ;;Do not remove dummy
+  (dummy 116)
```

Comment:
why is this entry needed, is it used somewhere?

Reply:
according to Hat Kid, removing 116 breaks the enum. If this is not the case, then it can be removed